.github/workflows/test-lang-java.yml

```diff
@@ -39,6 +39,7 @@ jobs:
         - '8'
         - '11'
         - '17'
+        - '18-ea'
```

Comment:
wondering if we can already target 19-ea here since it is already out

Reply:
we can't :-)
see the commit history in the PR. First I tried with 19-ea but it seems it is not yet available via the `setup-java` action.
I've tried it locally though and everything is fine after these changes!

Reply:
I wasn't able to find any distribution listed at https://github.com/actions/setup-java#supported-distributions that provides 19-ea at the moment.